test/screenshotter/ss_data.json

```diff
@@ -4,6 +4,7 @@
     "Baseline": "http://localhost:7936/test/screenshotter/test.html?m=a+b-c\\cdot d/e",
     "BasicTest": "http://localhost:7936/test/screenshotter/test.html?m=a",
     "BinomTest": "http://localhost:7936/test/screenshotter/test.html?m=\\dbinom{a}{b}\\tbinom{a}{b}^{\\binom{a}{b}+17}",
+    "Cases": "http://localhost:7936/test/screenshotter/test.html?m=f%28a%2Cb%29%3D\\begin{cases}a%2B1%26\\text{if%20}b\\text{%20is%20odd}\\\\a%26\\text{if%20}b%3D0\\\\a-1%26\\text{otherwise}\\end{cases}",
```

Comment:
That was copy&paste from my browser, I can certainly simplify this. I often think about whether we should have some non-JSON data structure here, so we can just write the formula without any escaping for URL or JavaScript. I'm not sure how to include the other parameters in such a setup, but we could certainly invent a suitable format. Perhaps something that looks like a simple TeX file, but gets parsed by JavaScript? In any case, that parsing imho only makes sense after #264 landed, to avoid the Python components in the mix. What do you think?